jolt-core/src/jolt/vm/registers.rs

```diff
@@ -650,26 +636,27 @@ impl<F: JoltField, ProofTranscript: Transcript> ReadWriteCheckingProof<F, ProofT
 
                                 // Write-checking sumcheck
                                 if !rd_wa[0][k].is_zero() || !rd_wa[1][k].is_zero() {
-                                    // Save a mult by multiplying by `eq_r_eval` sooner rather than later
-                                    let eq_r_eval = eq_r.get_coeff(k);
                                     let wa_eval_0 = if rd_wa[0][k].is_zero() {
                                         F::zero()
                                     } else {
-                                        let eval = eq_r_eval * rd_wa[0][k];
-                                        inner_sum_evals[0] += eval * (wv_evals[0] - val_j_r[0][k]);
-                                        eval
+                                        rd_wa[0][k]
```

Comment:
if you're going to do this you might as well just do
`let wa_eval_0 = rd_wa[0][k];`